### PR TITLE
fix: lowercase all columns in examples

### PR DESCRIPTION
**Pull request by betodealmeida**
### SUMMARY

This PR converts all column names in the new examples to lowercase, to ensure they work across different databases.

This is necessary because in Postgresql columns with mixed case should be double quoted, but MySQL interprets double quoted values as strings.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Using MySQL:

Screenshot_2021-01-14 COVID Vaccine Dashboard(1)

Screenshot_2021-01-14 Slack Dashboard

Screenshot_2021-01-14 Sales Dashboard

Screenshot_2021-01-14 Sales Dashboard(1)

Screenshot_2021-01-14 Video Game Sales

Screenshot_2021-01-14 Video Game Sales(1)

Screenshot_2021-01-14 FCC New Coder Survey 2018

Screenshot_2021-01-14 FCC New Coder Survey 2018(1)

Screenshot_2021-01-14 FCC New Coder Survey 2018(2)

Using Postgresql:

Screenshot_2021-01-14 COVID Vaccine Dashboard(2)

Screenshot_2021-01-14 Slack Dashboard(1)

Screenshot_2021-01-14 Sales Dashboard(2)

Screenshot_2021-01-14 Sales Dashboard(3)

Screenshot_2021-01-14 Video Game Sales(2)

Screenshot_2021-01-14 Video Game Sales(3)

Screenshot_2021-01-14 FCC New Coder Survey 2018(3)

Screenshot_2021-01-14 FCC New Coder Survey 2018(4)

Screenshot_2021-01-14 FCC New Coder Survey 2018(5)

Using Sqlite:

Screenshot_2021-01-14 COVID Vaccine Dashboard(3)

Note: the Slack dashboard is different in Sqlite because the "Big Number with Trendline" visualization does not sort by time, and Sqlite returns the data in random order. I'll file a ticket.

Screenshot_2021-01-14 Slack Dashboard(2)

Screenshot_2021-01-14 Sales Dashboard(4)

Screenshot_2021-01-14 Sales Dashboard(5)

Screenshot_2021-01-14 Video Game Sales(4)

Screenshot_2021-01-14 Video Game Sales(5)

Screenshot_2021-01-14 FCC New Coder Survey 2018(6)

Screenshot_2021-01-14 FCC New Coder Survey 2018(7)

Screenshot_2021-01-14 FCC New Coder Survey 2018(8)

### TEST PLAN

I installed Superset from scratch into 3 different databases, and tested the dashboards.

### ADDITIONAL INFORMATION

- [x] Has associated issue: https://github.com/apache/superset/issues/12528
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/examples/configs/charts/Filter_Segments.yaml
Comment (eschutho) [inline superset/examples/configs/charts/Filter_Segments.yaml:56]: small nit, do you want to capitalize Live?

**Review by eschutho — APPROVED** [review]
code looks good. I didn't do a visual test, but screenshots look good.. @yousoph do you want to test?

File: superset/examples/configs/charts/Filter_Segments.yaml
Comment (betodealmeida) [reply superset/examples/configs/charts/Filter_Segments.yaml:56]: Good point, but TBH we might need to come up with something better. @srinify what should be the label here? "Country currently living on"?

**Review by villebro — APPROVED** [review]

**Review by junlincc — COMMENTED** [review]
LGTM after merged.

**Pull request merged**
